tempodb/encoding/vparquet3/block_traceql.go

```diff
@@ -1710,7 +1711,7 @@ func (c *spanCollector) KeepGroup(res *parquetquery.IteratorResult) bool {
 			sp.durationNanos = durationNanos
 			sp.attributes[traceql.IntrinsicDurationAttribute] = traceql.NewStaticDuration(time.Duration(durationNanos))
 		case columnPathSpanName:
-			sp.attributes[traceql.IntrinsicNameAttribute] = traceql.NewStaticString(kv.Value.String())
+			sp.attributes[traceql.IntrinsicNameAttribute] = traceql.NewStaticString(unsafeToString(kv.Value.Bytes()))
```

Comment:
The parquet-go library is already doing an [unsafe cast inside .String()](https://github.com/parquet-go/parquet-go/blob/main/value.go#L467).  But it looks like it incurs an extra [switch on kind](https://github.com/parquet-go/parquet-go/blob/main/value.go#L766).  Is the performance overhead of the switch that much?  Now I'm  wondering if traceql.Static should store byte[] ultimately.

Reply:
This is the String() method it's calling which just does a string([]byte) conversion:

https://github.com/parquet-go/parquet-go/blob/1748e4b3ff043d23973d60df55b4f74b80c92edf/value.go#L751

Reply:
I tried something silly like: `((*parquet.Value)(&v)).String()` but it still went to the value receiver